java/pom.xml

```diff
@@ -870,7 +897,7 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-surefire-plugin</artifactId>
             <configuration>
-              <argLine>--add-opens=java.base/java.nio=ALL-UNNAMED</argLine>
+              <argLine>--add-opens=java.base/java.nio=org.apache.arrow.memory.core,ALL-UNNAMED</argLine>
```

Comment:
Just to be sure, do we still need ALL-UNNAMED?

Reply:
Yes, it's still needed. Netty utilizes it for example.